js/exportsms.js

```diff
@@ -529,4 +557,8 @@ var exportsms = function() {
     format: format
   };
 
-}();
\ No newline at end of file
+}();
+
+function isEmpty(obj){
```

Comment:
I might be wrong here, but would this place `isEmpty` in the global scope? Why not just place it in the function scope of `exportsms`?